layouts/shortcodes/added_in.html

```diff
@@ -0,0 +1,21 @@
+{{ $feature := .Get "feature" }}
+{{ $update := .Get "update" }}
+
+{{ with $.Site.GetPage "page" "updates" $update }}
+  <div class="notices note">
```

Comment:
It looks like there might be a way to reuse the shortcode here instead of defining the HTML manually, but it looks like some research into that is required, so for now this will do

Reply:
Yeah, I'd hoped to do so, but couldn't make it work.